backend/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.224.2/containers/dotnet/.devcontainer/base.Dockerfile
+
+# [Choice] .NET version: 6.0, 5.0, 3.1, 6.0-bullseye, 5.0-bullseye, 3.1-bullseye, 6.0-focal, 5.0-focal, 3.1-focal
+ARG VARIANT="6.0-bullseye-slim"
```

Comment:
Should we be using .NET version 5.0 here?

Reply:
I tried this first of all.  Found out that it wouldn't work for some reason.  It's backward compatible though.